.rscignore

```diff
@@ -7,3 +7,4 @@ data/national_provider_summary.csv
 README.html
 data/apprenticeships_data.csv
 data/apprenticeships_demographics.csv
+.hooks
```

Comment:
Not seen this be necessary before? What did you see that led to this?

Reply:
Some of the workflows were producing annoying messages around the commit hooks when sourcing the .Rprofile script (none of which is needed by GitHub Actions), so I made a few changes with the aim of reducing the messaging and optimising the workflows.